### PR TITLE
New landing page template

### PR DESCRIPTION
**Pull request by tjsilver**
## Why are you doing this?

We are testing whether a new design of landing page will provide any uplift.

[**Trello Card**](https://trello.com/c/bv0IWb3W/1562-implement-a-new-template-for-the-landing-page)

## Changes

* Control - original landing page
* new_template - new landing page

### Mobile
image

### Tablet (740px +)
image

### Desktop (980px +)
image

### Wide (1440px +)
image

### CODE REVIEW

File: support-frontend/assets/pages/contributions-landing/components/ContributionFormContainer.jsx [hunk 1]
```diff
@@ -14,6 +14,9 @@ import { type State } from '../contributionsLandingReducer';
 import { ContributionForm, EmptyContributionForm } from './ContributionForm';
 import { onThirdPartyPaymentAuthorised, paymentWaiting, setTickerGoalReached } from '../contributionsLandingActions';
 import type { IsoCountry } from 'helpers/internationalisation/country';
+import type { PaymentSecuritySecureTransactionGreyNonUKVariants, NewLandingPageTemplateTestVariants } from 'helpers/abTests/abtestDefinitions';
+import SecureTransactionIndicator from '../../../../assets/components/secureTransactionIndicator/secureTransactionIndicator';
```
Comment (jlieb10) [inline support-frontend/assets/pages/contributions-landing/components/ContributionFormContainer.jsx]:
```suggestion
import SecureTransactionIndicator from 'assets/components/secureTransactionIndicator/secureTransactionIndicator';
```

File: support-frontend/assets/pages/contributions-landing/contributionsLanding.jsx [hunk 2]
```diff
@@ -33,6 +34,7 @@ import ConsentBanner from '../../components/consentBanner/consentBanner';
 import SecureTransactionIndicator from '../../../assets/components/secureTransactionIndicator/secureTransactionIndicator';
```
Comment (jlieb10) [inline support-frontend/assets/pages/contributions-landing/contributionsLanding.jsx]:
```suggestion
import SecureTransactionIndicator from 'components/secureTransactionIndicator/secureTransactionIndicator';
```

File: support-frontend/assets/pages/contributions-landing/components/ContributionFormContainer.jsx [hunk 3]
```diff
@@ -14,6 +14,9 @@ import { type State } from '../contributionsLandingReducer';
 import { ContributionForm, EmptyContributionForm } from './ContributionForm';
 import { onThirdPartyPaymentAuthorised, paymentWaiting, setTickerGoalReached } from '../contributionsLandingActions';
 import type { IsoCountry } from 'helpers/internationalisation/country';
+import type { PaymentSecuritySecureTransactionGreyNonUKVariants, NewLandingPageTemplateTestVariants } from 'helpers/abTests/abtestDefinitions';
+import SecureTransactionIndicator from 'assets/components/secureTransactionIndicator/secureTransactionIndicator';
```
Comment (jlieb10) [inline support-frontend/assets/pages/contributions-landing/components/ContributionFormContainer.jsx]:
```suggestion
import SecureTransactionIndicator from components/secureTransactionIndicator/secureTransactionIndicator';
```

**Comment by ionamckendrick** [comment]
Would you mind confirming if we are running this out to everyone (MINUS US) as a test first? And then we add them in later? Due to the end of year campaign.

**Comment by jlieb10** [comment]
@ionamckendrick the spec changed per a conversation between me and Victoria - we will be rolling this out to 100% of the audience (US included) -- the US does not to be excluded any more because they only have EOY copy on the landing page, rather than a whole design that needs to be considered

**Pull request merged**